### PR TITLE
chore: support ivy based partial compilation for @ngqp/core

### LINKED ISSUE #224: Proper support of angular 14
Angular ngcc processing ngqp as view engine:
```
Processing legacy "View Engine" libraries:
- @ngqp/core [fesm2015/esm2015] (https://github.com/TNG/ngqp.git)
- @ngqp/core [esm2015/esm2015] (https://github.com/TNG/ngqp.git)
- @ngqp/core [main/umd] (https://github.com/TNG/ngqp.git)

```
**What's your idea?**

Save time for ngcc processing
**Describe the solution you'd like**

Could you please update dependencies to 14 angular in package json and ng-packagr and after fix all stuff update your lib (yes it's breaking change). Because it should be compiled properly against ivy, and also check parameter `compilationMode: 'partial'` for publishing.
**Additional context**

### PR DESCRIPTION
**Pull request by pumano**
Library should be compiled using "compilationMode": "partial" for not using ngcc
Also default target for 14 angular is es2020
Update browserlist to actual due to bugs with old data
Change legacy "fullTemplateTypeCheck" to actual "strictTemplates"

BREAKING CHANGE: due to partial ivy compilation it's not supported by angular before 11.1 so bump version to 15

closes #224

This pull request relates to or closes the following issues:

I have verified that…
- [x] the commits in this pull request follow the [conventionalcommits](https://www.conventionalcommits.org) pattern
- [ ] tests have been updated or added
- [ ] documentation has been updated to reflect the changes made

### CODE REVIEW

File: package.json
Comment (Airblader) [inline package.json:32]: This also updates ngqp-demo. What about the issues with Webpack? For me this doesn't run locally.

File: projects/ngqp/core/package.json
Comment (Airblader) [inline projects/ngqp/core/package.json:3]: Please undo this change, this is done during release. Also, move the

> BREAKING CHANGE: due to partial ivy compilation it's not supported by angular before 11.1 so bump version to 15

comment into the commit message, because that is where conventional commits need it. But leave out the "version 15" there, that is decided during release. Just an explanation of why it is breaking: the dependencies bump. 🙂

**Review by Airblader — CHANGES_REQUESTED** [review]
Thanks for the PR, I left a couple comments / questions. Please also make sure to sign your commit(s) for the DCO.

File: package.json
Comment (pumano) [reply package.json:32]: That is because you use `raw-loader` for load ts files as string for examples. That raw-loader is removed from webpack 5 and angular cli 14 uses webpack 5

Comment (Airblader) [reply package.json:32]: I'm well aware of the problem, but I don't understand the change. If this simply doesn't work we of course can't merge it.

**Comment by pumano** [comment]
@Airblader OK I revert my work, just changed to `ivyEnabled` to true

**Comment by Airblader** [comment]
Can you please try to get the sign-off to work? The DCO is a company requirement.

The commit message currently also doesn't really make much sense; it talks about reverting a commit.

**Pull request closed without merging**
